### PR TITLE
Multiple Networks support (part 2)

### PR DESCRIPTION
**Pull request by Jack-Works**

### CODE REVIEW

File: src/components/shared/SelectPeople.tsx [hunk 1]
```diff
@@ -131,38 +161,39 @@ export function SelectPeopleUI(props: SelectPeopleUI) {
...
+                {showSelectNone && SelectNoneButton}
+            </Box>
+            <Box flex={1}>
+                <List dense>
+                    {listBeforeSearch.length > 0 && listBeforeSearch.length === 0 && (
+                        <ListItem>
+                            <ListItemText primary={geti18nString('not_found')} />
+                        </ListItem>
+                    )}
+                    {listAfterSearch.map(PeopleListItem)}
```
Comment (guanbinrui) [inline src/components/shared/SelectPeople.tsx]: Why can't I select person with `nickname`? Cause we display person with `nickname` but can't search by that.

File: src/components/shared/SelectPeople.tsx [hunk 2]
```diff
@@ -131,38 +161,39 @@ export function SelectPeopleUI(props: SelectPeopleUI) {
...
-                </>
-            )}
-        </>
+            <Box display="flex">
+                {showSelectAll && SelectAllButton}
+                {showSelectNone && SelectNoneButton}
+            </Box>
+            <Box flex={1}>
+                <List dense>
+                    {listBeforeSearch.length > 0 && listBeforeSearch.length === 0 && (
```
Comment (guanbinrui) [inline src/components/shared/SelectPeople.tsx]: It's unreachable. If on purpose maybe leave a comment?

File: src/social-network/defaults/injectComments.tsx [hunk 3]
```diff
@@ -16,28 +16,28 @@ export function injectPostCommentsDefault(config: injectPostCommentsDefaultConfi
     return function injectPostComments(this: SocialNetworkUI, current: PostInfo) {
         const selector = current.commentsSelector
         const commentWatcher = new MutationObserverWatcher(selector, current.rootNode)
-            .useForeach(commentNode => {
-                const commentRef = new ValueRef(commentNode.current.innerText)
+            .useForeach((commentNode, key, meta) => {
+                const commentRef = new ValueRef(commentNode.innerText)
```
Comment (Misaka-0x447f) [inline src/social-network/defaults/injectComments.tsx]: property 'innerText' does not exist on 'commentNode'. did you forgot to reference in 'current'?

File: src/social-network/defaults/injectComments.tsx [hunk 4]
```diff
@@ -16,28 +16,28 @@ export function injectPostCommentsDefault(config: injectPostCommentsDefaultConfi
...
                 const needZipDefault = () => {
-                    commentNode.current.style.whiteSpace = 'nowrap'
-                    commentNode.current.style.overflow = 'hidden'
+                    commentNode.style.whiteSpace = 'nowrap'
+                    commentNode.style.overflow = 'hidden'
                 }
-                const injectionPointDefault = () => commentNode.afterShadow
+                const injectionPointDefault = () => meta.afterShadow
```
Comment (Misaka-0x447f) [inline src/social-network/defaults/injectComments.tsx]: property 'afterShadow' does not exist on 'meta'.

**Comment by Misaka-0x447f** [comment]
More TS Error exists so cannot perform merge action now.

**Comment by Jack-Works** [comment]
Actually you need to yarn again

File: src/components/InjectedComponents/DecryptedPost.tsx [hunk 5]
```diff
@@ -41,7 +46,7 @@ function DecryptPostSuccess({ data, people, ...props }: DecryptPostSuccessProps)
                 <>
                     {ShareMenu}
                     {geti18nString('decrypted_postbox_title')}
-                    <Box flex={1} />
+                    <Box flex={1}/>
```
Comment (Jack-Works) [inline src/components/InjectedComponents/DecryptedPost.tsx]: Your prettier breaks again?

Comment (Misaka-0x447f) [reply src/components/InjectedComponents/DecryptedPost.tsx]: ok actually no prettier running during this change. prettier only run during an npm script running.

**Pull request merged**